Purchases/SubscriberAttributes/RCSubscriberAttribute.m

```diff
@@ -33,14 +33,14 @@ @interface RCSubscriberAttribute ()
 
 @implementation RCSubscriberAttribute
 
-- (instancetype)initWithKey:(NSString *)key value:(NSString *)value {
+- (instancetype)initWithKey:(NSString *)key value:(nullable NSString *)value {
```

Comment:
the nullables in this file shouldn't be needed if we make sure that we never use a nil value, see my comment in `RCSubscriberAttributesManager.m`

Reply:
Removed after applying the change to use valueOrEmpty.